tests/c-tests/test-simple-induction-variable.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int main()
+{
+  int i = 0;
+  do {
+    i = 1 + i;
+  } while (i < 5);
+
+  return 0;
+}
```

Comment:
The test is a bit useless as it is currently not automatically tested. I rather suggest that you add a unit test that tests e.g. your FindInductionVariable() method.